gnome-initial-setup/gis-driver.c

```diff
@@ -246,6 +300,23 @@ gis_driver_activate (GApplication *app)
 }
 
 static void
+window_realize_cb (GtkWidget *widget, gpointer user_data)
```

Comment:
Nitpick: one parameter per line. You can also just have the driver there instead of the gpointer